tests/test_gates_abstract.py

```diff
@@ -228,10 +228,13 @@ def test_one_qubit_rotations_controlled_by(gatename, params):
     assert gate.parameters == params
 
 
-def test_cnot_and_cz_init():
+def test_cnot_and_cy_and_cz_init():
     gate = gates.CNOT(0, 1)
     assert gate.target_qubits == (1,)
     assert gate.control_qubits == (0,)
+    gate = gates.CZ(4, 7)
```

Comment:
```suggestion
    gate = gates.CY(4, 7)
```

is this supposed to be a CY?